lib/tlTimeline/RenderOptionsInline.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
The new .h files should be added to the CMakeLists,txt so that CMake knows about them.